src/Spryker/Zed/AppPayment/Business/Payment/Webhook/WebhookHandler.php

```diff
@@ -88,4 +97,20 @@ private function getTransactionIdOrFail(WebhookRequestTransfer $webhookRequestTr
 
         throw new InvalidArgumentException(MessageBuilder::getRequestTransactionIdIsMissingOrEmpty());
     }
+
+    private function validateWebhookRequest(
+        WebhookRequestTransfer $webhookRequestTransfer,
+        WebhookResponseTransfer $webhookResponseTransfer
+    ): WebhookResponseTransfer {
+        $appConfigTransfer = $webhookRequestTransfer->getAppConfig();
+
+        if (!$appConfigTransfer instanceof AppConfigTransfer || $appConfigTransfer->getStatus() === AppKernelConfig::APP_STATUS_DISCONNECTED) {
```

Comment:
if appConfigTransfer is empty, it should be another error I guess. now we ignore it. or is it for phpstan?

Reply:
When there is no config the code that tries to add it will fail already with an exception. This check is for PHPStan and I don't want to use OrFail here.